app/javascript/packages/modal/README.md

```diff
@@ -0,0 +1,32 @@
+# `@18f/identity-modal`
+
+Custom element for a modal dialog.
+
+## Usage
+
+Importing the element will register the `<lg-modal>` custom element:
+
+```ts
+import '@18f/identity-modal/modal-element';
+```
+
+The custom element will implement the copying behavior, but all markup must already exist.
```

Comment:
"copying" behavior? is that correct here?

Reply:
> "copying" behavior? is that correct here?

😂  Caught me copy-pasting!

Reply:
Fixed in 80665c3d5.